js/apps/system/_admin/aardvark/APP/frontend/js/views/settingsView.js

```diff
@@ -82,7 +82,7 @@
                 arangoHelper.arangoError('Collection error: ' + data.responseJSON.errorMessage);
               } else {
                 arangoHelper.arangoNotification('Collection: ' + 'Successfully changed.');
-                window.App.navigate('#cSettings/' + newname, {trigger: true});
+                window.App.navigate('#cSettings/' + encodeURIComponent(newname), {trigger: true});
```

Comment:
Maybe we can re-use `encodeHelper` ?

Reply:
we haven't used any react utils in backbone yet, and this code located awkardly too

import will look like this:

`import {encodeHelper} from '../../../react/src/utils/encodeHelper';`

If you think this is a good idea, we can do it, but I'd recommend just keeping what we have, as this is simple enough